edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -235,6 +240,16 @@ public static async Task<int> MainAsync(IConfiguration configuration)
                 return 1;
             }
 
+            // Initialize metrics
+            var metricsConfig = new MetricsConfig(true, MetricsListenerConfig.Create(configuration));
+            var metricsProvider = metricsConfig.Enabled
+                             ? new MetricsProvider("edgeagent", iothubHostname, deviceId)
```

Comment:
Please change `"edgeAgent"` to `configuration.GetValue<string>(Constants.IoTEdgeAgentProductInfoIdentifier)`

Reply:
It will have to be ```configuration.GetValue<string>(Constants.IoTEdgeAgentProductInfoIdentifier).ToLower()``` since the namespace shouldn't have caps.